src/utils/validateIdentifierInputs.ts

```diff
@@ -28,10 +28,13 @@ export const validateIdentifierInputs = (
     if (textInputType === "NUMBER" && isNaN(Number(value))) {
       throw new Error(ERROR_MESSAGE.INVALID_IDENTIFIER_INPUT);
     }
+
+    const trimmedValue = value.trim();
```

Comment:
we no longer need to trim here since it's already trimmed in `cleanIdentifierInputs`!! can remove this